libraries/testing/tester.cpp

```diff
@@ -1227,6 +1227,17 @@ namespace eosio { namespace testing {
 
       std::vector<builtin_protocol_feature_t> ordered_builtins;
       for( const auto& f : builtin_protocol_feature_codenames ) {
+         // Before deferred trxs feature is fully disabled, existing tests involving
+         // deferred trxs need to be exercised to make sure existing behaviors are
+         // maintained. Excluding DISABLE_DEFERRED_TRXS_STAGE_1 and DISABLE_DEFERRED_TRXS_STAGE_2
+         // from full protocol feature list such that existing tests can run.
+         // Remove the following exclusion once eferred trxs feature is fully disabled
+         // and tests are adapted.
+         std::string codename = f.second.codename;
+         if( codename.compare( "DISABLE_DEFERRED_TRXS_STAGE_1" ) == 0 || codename.compare( "DISABLE_DEFERRED_TRXS_STAGE_2" ) == 0 ) {
+            continue;
+         }
```

Comment:
Instead of this I think it would be better to create a new `setup_policy` that is everything but these two protocol features and use that for tests that need it. Then `preactivate_all_builtin_protocol_features()` means *all*.

Also why `compare` instead of just `==`?

Reply:
I was thinking about introducing a new specific policy too. But when we finally remove the deferred trxs, we need to update a number of places to change the policy back to `all`, instead of only changing one place.

Either way is fine with me. 

Yes, I will change to use `==` for more natural looking. Thanks.

Reply:
I have thought it about more. It is better to let `preactivate_all_builtin_protocol_features` mean really `all`. Will change the code. Thanks.